cmd/podman/containers/create.go

```diff
@@ -292,11 +292,10 @@ func createPodIfNecessary(s *specgen.SpecGenerator) (*entities.PodCreateReport,
 		return nil, errors.Errorf("new pod name must be at least one character")
 	}
 	createOptions := entities.PodCreateOptions{
-		Name:  podName,
-		Infra: true,
-		Net: &entities.NetOptions{
-			PublishPorts: s.PortMappings,
-		},
+		Name:          podName,
+		Infra:         true,
+		Net:           netOpts,
+		CreateCommand: os.Args,
```

Comment:
Probably best to leave this empty - it's mainly used for `podman generate systemd --new`, and it could have misleading results there

Reply:
I believe it is correct to set here. I checked `podman generate systemd --new` and it throws an error that you have to use `podman pod create`. This can later be used to implement this.

Reply:
@vrothberg Thoughts here?

Reply:
It looks good to me. As @Luap99 mentioned, `generate systemd` requires a pod to be generated via `pod create` and will error out otherwise (see https://github.com/containers/podman/blob/master/pkg/systemd/generate/pods.go#L265).

Setting it here will be more accurate as `generate systemd` will otherwise error out with "no create command found" which would be less informative than "pod does not appear to be created via `podman pod create`"

Reply:
Alright. We should be good to merge after a rebase, then.